docs/getting-started.md

```diff
@@ -69,3 +69,22 @@ If you've authenticated as a given user, you can query their details directly:
 ```
 var user = await client.User.Current();
 ```
+
+### Too Much of a Good Thing: Dealing with API Rate Limits
+Like any popular API, Github needs to throttle some requests. The OctoKit.NET client allows you to get some insight into how many requests you have left and when you can start making requests again. It does this via the `ApiInfo` object and the `GetLastApiInfo()` method.
```

Comment:
I think it would be better to link to [this page](https://developer.github.com/v3/#rate-limiting) since it provides info about rate limits etc, rather than the link you had (which is the doc for the actual "Get Rate Limit" API call)